recipes/wg-blimp/build.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
you can get rid of this file by using https://conda-forge.org/docs/meta.html?highlight=pip%20install#use-pip

Reply:
Updated to remove build.sh